moodlecli/utils.py

```diff
@@ -150,73 +150,28 @@ def setup_duplicate_course(
     moodle_client, base_course_id, coursedata, instructor_role_id,
     student_role_id
 ):
-    """Setup a new course using a base and data from bulk CSV"""
     # Retrieve or create teacher account
     instructor_user_id = create_or_get_user(
         moodle_client,
         coursedata[CSV_INST_FNAME],
         coursedata[CSV_INST_LNAME],
         coursedata[CSV_INST_EMAIL],
-        coursedata[CSV_INST_AUTH]
-    )
-    try:
-        # Create a duplicate course using the base course
-        new_course = moodle_client.copy_course(
-            base_course_id,
-            coursedata[CSV_COURSE_NAME],
-            coursedata[CSV_COURSE_SHORTNAME],
-            coursedata[CSV_COURSE_CATEGORY]
+        coursedata[CSV_INST_AUTH],
         )
-        new_course_id = new_course["id"]
-    except (ConnectionError, Timeout):
-        # Implement work around for Global Accelerator timeout of 340s
-        # (refer to https://github.com/openstax/k12/issues/316 for details)
-        print("Remote disconnected during course copy!")
-        print(f"Polling for course {coursedata[CSV_COURSE_SHORTNAME]}...")
-        course_found = False
-        while not course_found:
-            sleep(30)
-            print("Querying current courses")
-            data = moodle_client.get_course_by_shortname(
-                coursedata[CSV_COURSE_SHORTNAME])
-            if len(data['courses']) == 0:
-                continue
-            new_course_id = data["courses"][0]['id']
-            print(f"Found copy: Course ID {new_course_id}")
-            course_found = True
-
-    # Enrol teacher user as a course instructor
-    moodle_client.enrol_user(
-        new_course_id,
+    """Setup a new course using a base and data from bulk CSV"""
+    course = course_creation(
```

Comment:
Minor: Can we move this function to be closer to `setup_duplicate_course` since they're related?